### ISSUE TITLE
Dermatologist-level classification of skin cancer with deep neural networks

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#207]
https://doi.org/10.1038/nature21056 (DOI still processing, http://www.nature.com/nature/journal/vaop/ncurrent/full/nature21056.html)

> Skin cancer, the most common human malignancy1, 2, 3, is primarily diagnosed visually, beginning with an initial clinical screening and followed potentially by dermoscopic analysis, a biopsy and histopathological examination. Automated classification of skin lesions using images is a challenging task owing to the fine-grained variability in the appearance of skin lesions. Deep convolutional neural networks (CNNs)4, 5 show potential for general and highly variable tasks across many fine-grained object categories6, 7, 8, 9, 10, 11. Here we demonstrate classification of skin lesions using a single CNN, trained end-to-end from images directly, using only pixels and disease labels as inputs. We train a CNN using a dataset of 129,450 clinical images—two orders of magnitude larger than previous datasets12—consisting of 2,032 different diseases. We test its performance against 21 board-certified dermatologists on biopsy-proven clinical images with two critical binary classification use cases: keratinocyte carcinomas versus benign seborrheic keratoses; and malignant melanomas versus benign nevi. The first case represents the identification of the most common cancers, the second represents the identification of the deadliest skin cancer. The CNN achieves performance on par with all tested experts across both tasks, demonstrating an artificial intelligence capable of classifying skin cancer with a level of competence comparable to dermatologists. Outfitted with deep neural networks, mobile devices can potentially extend the reach of dermatologists outside of the clinic. It is projected that 6.3 billion smartphone subscriptions will exist by the year 2021 (ref. 13) and can therefore potentially provide low-cost universal access to vital diagnostic care.

The emphasis on large, high-quality training datasets in the abstract should be relevant for our discussion section.

**Comment by gwaygenomics** [comment]
Article summarized in Nature Reviews Cancer [Research Highlight](https://doi.org/10.1038/nrc.2017.10)
